setup.sh

```diff
@@ -58,90 +141,85 @@ else
 fi
 
 # Check if the backend module can be removed
+if ! whiptail --backtitle "New CARDS repository setup" --title "Modules setup" --yesno "Does the project have backend code?" 8 78
+then
+  git rm -rf backend
+  sed -i -e '/backend/,+3d' feature/src/main/features/feature.json
+  sed -i -e '/backend/d' pom.xml
+fi
 
-echo
-echo "Does the project have backend code?"
-HAS_BACKEND=maybe
-until [[ $HAS_BACKEND == "yes" || $HAS_BACKEND == "no" ]]
-do
-  read -e -p "[yes|no] " -i "no" HAS_BACKEND
+./get_cards_platform_jars.sh ${CARDS_VERSION} || exit -1
+# Ask about the permission scheme
 
-  if [[ $HAS_BACKEND != "yes" && $HAS_BACKEND != "no" ]]
+# Get the list of features already included in the base distribution
+# Then get the list of all features known, excluding the cards4* projects
+# Then subtract included features from the list of all known features
```

Comment:
Does this comment belong here? It's the `permissionlist` list that is being built as opposed to the `featurelist` which is built later in this script and described by the same comment https://github.com/data-team-uhn/cards-project-template/pull/1/files#diff-4209d788ad32c40cbda3c66b3de47eefb929308ca703bb77a6382625986add17R177-R179

Reply:
It used to be correct, but it doesn't reflect the current code.

Reply:
Removed.